src/components/application_manager/include/application_manager/commands/command_impl.h

```diff
@@ -183,6 +200,16 @@ class CommandImpl : public Command {
    */
   bool CheckSyntax(const std::string& str, bool allow_empty_line = false) const;
 
+  /**
+   * @brief Checks HMI result code for single RPC
+   * @param result_code contains result code from HMI response
+   * @param interface to check availability
+   * @return true if result code complies to successful result codes,
+   * false otherwise.
+   */
+  bool IsHMIResultSuccess(hmi_apis::Common_Result::eType result_code,
```

Comment:
IsHMIResultSuccess at line 88 is free function, but this IsHMIResultSuccess is method of CommandImpl class, because here we need access to application_manager_ member of CommandImpl class. So we cannot combine them that way.